src/wrapped/wrappedsdl2.c

```diff
@@ -588,6 +591,104 @@ EXPORT int my2_SDL_snprintf(x86emu_t* emu, void* buff, uint32_t s, void * fmt, v
     #endif
 }
 
+static int get_sdl_priv(x86emu_t* emu, const char *sym_str, void **w, void **f)
+{
+    sdl2_my_t *my = (sdl2_my_t *)emu->context->sdl2lib->priv.w.p2;
+
+    #define GO(sym, _w) \
+        if (strcmp(#sym, sym_str) == 0) \
+        { \
+            *w = _w; \
+            *f = dlsym(emu->context->sdl2lib->priv.w.lib, #sym); \
+            return *f != NULL; \
+        }
+    #define GO2(sym, _w, sym2) \
+        if (strcmp(#sym, sym_str) == 0) \
+        { \
+            *w = _w; \
+            *f = dlsym(emu->context->sdl2lib->priv.w.lib, #sym2); \
+            return *f != NULL; \
+        }
+    #define GOM(sym, _w) \
+        if (strcmp(#sym, sym_str) == 0) \
+        { \
+            *w = _w; \
+            *f = dlsym(emu->context->box86lib, "my2_"#sym); \
+            return *f != NULL; \
+        }
+    #define GOS(sym, _w) GOM(sym, _w)
+
+    #define DATA
+    if(0);
```

Comment:
What's the use of this `if`?

Reply:
Part of a change that got lost during debugging, it was supposed to allow the GO macros to be `else if` rather than a chain of endless `if`